### PR TITLE
adding article to forms docs regarding removing built in fields

### PR DESCRIPTION
**Pull request by alina-tincas**
## Description

_What did you add/update/change?_
Added new article page on forms docs regarding excluding/removing built-in fields as per this PR: #5436 
## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
Umbraco Forms, all versions

## Deadline (if relevant)
anytime
_When should the content be published?_
anytime

### CODE REVIEW

File: 10/umbraco-forms/SUMMARY.md [hunk 1]
```diff
@@ -17,8 +17,8 @@
   * [Overview Of The Field Types](editor/creating-a-form/fieldtypes/README.md)
     * [Date](editor/creating-a-form/fieldtypes/date.md)
     * [File Upload](editor/creating-a-form/fieldtypes/fileupload.md)
-    * [reCAPTCHA V2](editor/creating-a-form/fieldtypes/recaptcha2.md)
-    * [reCAPTCHA V3](editor/creating-a-form/fieldtypes/recaptcha3.md)
+    * [ReCAPTCHA V2](editor/creating-a-form/fieldtypes/recaptcha2.md)
```
Comment (AndyButland) [inline 10/umbraco-forms/SUMMARY.md:20]: I think these were correct as they were - as "reCAPTCHA" is the brand name (see [here](https://www.google.com/recaptcha/about/)).  Not sure what the title is on the page they link to, but that should also use "reCAPTCHA" and not "ReCAPTCHA".

File: 10/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Excluding a Built-in Field
+
+Umbraco Forms comes with some built-in fields however it is possible to exclude/remove them if necessary.
+There might some use cases where you have no use for file upload and don't want editors using them. Or it could be there's some new functionality you need you want to build yourself.
```
Comment (AndyButland) [inline 10/umbraco-forms/developer/extending/excluding-a-built-in-field.md:4]:
```suggestion
There might some use cases where you have no use for file upload and don't want editors using them. Or perhaps you want to remove a field to replace it with one with enhanced functionality that you build yourself.
```

File: 11/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Excluding a Built-in Field
+
+Umbraco Forms comes with some built-in fields however it is possible to exclude/remove them if necessary.
+There might some use cases where you have no use for file upload and don't want editors using them. Or it could be there's some new functionality you need you want to build yourself.
```
Comment (AndyButland) [inline 11/umbraco-forms/developer/extending/excluding-a-built-in-field.md:4]:
```suggestion
There might some use cases where you have no use for file upload and don't want editors using them. Or perhaps you want to remove a field to replace it with one with enhanced functionality that you build yourself.
```

File: 12/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Excluding a Built-in Field
+
+Umbraco Forms comes with some built-in fields however it is possible to exclude/remove them if necessary.
+There might some use cases where you have no use for file upload and don't want editors using them. Or it could be there's some new functionality you need you want to build yourself.
```
Comment (AndyButland) [inline 12/umbraco-forms/developer/extending/excluding-a-built-in-field.md:4]:
```suggestion
There might some use cases where you have no use for file upload and don't want editors using them. Or perhaps you want to remove a field to replace it with one with enhanced functionality that you build yourself.
```

File: 12/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Example
+
+The following class shows how to exclude built-in field types using a custom composer. The `Password`, `Recaptcha2` and `RichText` field types (or "answers") will no longer be available for selection when creating a form in the backoffice.
+
+```csharp
+using Umbraco.Forms.Core.Providers;
+using Umbraco.Forms.Core.Providers.FieldTypes;
+namespace MyNamespace
```
Comment (AndyButland) [inline 12/umbraco-forms/developer/extending/excluding-a-built-in-field.md:13]:
```suggestion

namespace MyNamespace
```

File: 11/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+## Example
+
+The following class shows how to exclude built-in field types using a custom composer. The `Password`, `Recaptcha2` and `RichText` field types (or "answers") will no longer be available for selection when creating a form in the backoffice.
+
+```csharp
+using Umbraco.Forms.Core.Providers;
+using Umbraco.Forms.Core.Providers.FieldTypes;
+namespace MyNamespace
```
Comment (AndyButland) [inline 11/umbraco-forms/developer/extending/excluding-a-built-in-field.md:13]:
```suggestion

namespace MyNamespace
```

File: 10/umbraco-forms/developer/extending/excluding-a-built-in-field.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+## Example
+
+The following class shows how to exclude built-in field types using a custom composer. The `Password`, `Recaptcha2` and `RichText` field types (or "answers") will no longer be available for selection when creating a form in the backoffice.
+
+```csharp
+using Umbraco.Forms.Core.Providers;
+using Umbraco.Forms.Core.Providers.FieldTypes;
+namespace MyNamespace
```
Comment (AndyButland) [inline 10/umbraco-forms/developer/extending/excluding-a-built-in-field.md:13]:
```suggestion

namespace MyNamespace
```

**Review by AndyButland — APPROVED** [review]
Looks good to me - I made a few suggestions you can consider.

**Comment by alina-tincas** [comment]
Thank you for the suggestions, I have approved them! Am merging this in then 😊

**Pull request merged**